doc/Output JSON.json

```diff
@@ -367,12 +396,7 @@
       }, {
         "title": "Ongoing Drugs",
```

Comment:
no such section in ABDM profile. there is a medication section, all medication statement or medication request must be in that section. Please also use SNOMED code 721912009 for the section code